src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -122,6 +122,17 @@ public function load(array $configs, ContainerBuilder $container)
             $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
         }
 
+        if ($container->fileExists($dir = $container->getParameter('kernel.project_dir').'/templates', false)) {
+            $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($dir));
```

Comment:
Instead of harcoding the `templates/` directory here, what about adding a new configuration setting that would default to `templates/` and use it here? Should be easy enough.